### PR TITLE
fix(output): handle `--unix-timestamp` flag

### LINKED ISSUE #4077: [Bug]: Unix timestamp not working
### Steps to Reproduce

Run prowler using `--unix-timestamp`

### Expected behavior

Outputs should have the unix timestamp format

### Actual Result with Screenshots or Logs

image

### How did you install Prowler?

Cloning the repository from github.com (git clone)

### Environment Resource

locally

### OS used

all

### Prowler version

4.1

### Pip version

latest

### Context

_No response_

### PR DESCRIPTION
**Pull request by pedrooot**
### Context

Fixes #4077

### Description

Modify FindingOutput class to make timestamp an int or datetime.

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: prowler/lib/outputs/common_models.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ class FindingOutput(BaseModel):
     """
 
     auth_method: str
-    timestamp: datetime
+    timestamp: Union[int, datetime]
```
Comment (jfagoagas) [inline prowler/lib/outputs/common_models.py:32]: Can you add a test creating a `FindingOutput` with the unix timestamp?

**Review by jfagoagas — CHANGES_REQUESTED** [review]
Great catch @pedrooot 👏

File: prowler/lib/outputs/common_models.py (hunk 1 above)
Comment (pedrooot) [reply prowler/lib/outputs/common_models.py:32]: Sure!

Comment (pedrooot) [reply prowler/lib/outputs/common_models.py:32]: Done ✅

**Review by jfagoagas — APPROVED** [review]
Great work @pedrooot 👏

**Pull request merged**